src/hooks/useBeaconchainData.tsx

```diff
@@ -0,0 +1,64 @@
+// Import libraries
+import { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+// Utilities + types
+import {
+  fetchTotalStakeAndAPR,
+  FetchTotalStakeAndAPRResponse,
+} from '../utils/fetchTotalStakeAndAPR';
+import {
+  fetchTotalValidators,
+  FetchTotalValidatorsResponse,
+} from '../utils/fetchTotalValidators';
+import { NetworkState } from '../pages/Landing/NetworkStatus';
+
+export const useBeaconchainData = () => {
+  const { locale } = useIntl();
+  const [state, setState] = useState<NetworkState>({
+    amountEth: '',
+    apr: '',
+    totalValidators: '',
+    status: 0,
+  });
+
+  useEffect(() => {
+    // Number formatters
+    const formatPercent = (value: number): string =>
+      new Intl.NumberFormat(locale, {
+        style: 'percent',
+        minimumSignificantDigits: 3,
+        maximumSignificantDigits: 3,
+      }).format(value);
+    const formatNumber = (value: number): string =>
+      new Intl.NumberFormat(locale).format(value);
```

Comment:
Would suggest moving these functions to the `utils` function. Not a blocker though